File: Ad-Hoc/URI 1893.hs
main :: IO ()
main = do
    line <- getLine
    let (inicio:final:_) = map (read :: String -> Int) $ words line

    if inicio >= 0 && final <= 2
        then putStrLn $ "nova"
    else if final > inicio && final <= 96
        then putStrLn $ "crescente"
    else if inicio >= final && final <= 96
        then putStrLn $ "minguante"
    else
        putStrLn $ "cheia"

File: Ad-Hoc/URI 1893.clj
(require '[clojure.string :as str])

(def line (str/split (read-line) #" "))
(def inicio (Integer/parseInt (get line 0)))
(def final (Integer/parseInt (get line 1)))

(cond
    (and (>= inicio 0) (<= final 2)) 
        (println "nova")
    
    (and (> final inicio) (<= final 96)) 
        (println "crescente")
    
    (and (>= inicio final) (<= final 96)) 
        (println "minguante")
    
    :else 
        (println "cheia")   
)